packages/joint-core/types/joint.d.ts

```diff
@@ -1992,7 +1992,9 @@ export namespace dia {
 
         protected insertView(cellView: CellView, isInitialInsert: boolean): void;
 
-        protected detachView(cellView: CellView): void;
+        protected _hideCellView(cellView: CellView): void;
```

Comment:
The naming is different from all other methods we have currently

Reply:
There are two reasons I chose this name:
- Some methods are designed to work with any `mvc.View` (such as requestViewUpdate()), while others apply specifically to `CellView`. It’s important to distinguish between these cases clearly.
- I use the underscore (`_`) to indicate that this method is not public, and also that it should never be called independently, it only hides the view without updating internal structures.

Reply:
Ok, I think if it is not supposed to be called, then underscore is fine in this case. We have more untyped methods with underscore currently, we can expose them later in the same vein.